src/services/userService.js

```diff
@@ -0,0 +1,38 @@
+import Api from "./api";
+
+const UserService = {
+  getAllUsers: async () => {
+    try {
+      const response = await Api.get("/api/users");
```

Comment:
Appreciation: It's good that you are using a common interceptor to map the token to API req